app/frontend/components/module_assistant/module_assistant.js

```diff
@@ -0,0 +1,228 @@
+  user-select: none;
+}
+
+#module-settings-assistant > .container {
+  height: 100%;
+  box-sizing: border-box;
+  padding-top: 1em;
+}
+
+#module-settings-assistant-init {
+  width: 100%;
+  height: 100%;
+  min-height: inherit;
+  box-sizing: border-box;
+  display: grid;
+  place-items: center;
+  text-align: center;
+}
+
+#module-settings-assistant-init > section {
+  padding: 2em 5em; 
+  border-radius: 5px;
+  min-width: 50%;
+  background: var(--widget-bg-color);
+}
+
+
+#module-settings-assistant-init > .module-assistant-type-buttons {
+  align-self: start;
+  display: grid;
+  grid-auto-flow: column;
+  gap: 5em;
+}
+
+#module-settings-assistant-init button {
+  height: 2.2em;
+  padding: 0 2em;
+}
+
+#module-settings-assistant section.scrollable {
+  overflow-y: auto;
+}
+#module-settings-assistant .content .body {
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+}
+
+</style>
+
+<div id="module-settings-assistant" style="display: none;">
+  <div class="container">
+  <div id="module-settings-assistant-init">
+    <section>
+      <p id="module-settings-assistant-internet-usage" style="margin-bottom: 2em;"></p>
+    </section>
+    <div class="module-assistant-type-buttons">
+      <button id="add-modules-button" class="fg-button ui-corner-all ui-state-default ui-state-disabled"></button>
+      <button id="remove-modules-button" class="fg-button ui-corner-all ui-state-default ui-state-disabled"></button>
+    </div>
+  </div>
+
+  <assistant-steps-add></assistant-steps-add>
+
+  <assistant-steps-remove></assistant-steps-remove>
+  </div>
+</div>
+`;
+
+class ModuleAssistant extends HTMLElement{
+  constructor() {
+    super();
+    this._initialized = false;
+  }
+
+  connectedCallback() {
+    if (this._initialized) {
+      return;
+    }
+
+    this.appendChild(template.content);
+
+    const addButton = document.querySelector('#add-modules-button');
+    addButton.addEventListener('click', async () => this._startAddModuleAssistant());
+
+    const removeButton = document.querySelector('#remove-modules-button');
+    removeButton.addEventListener('click', async () => this._startRemoveModuleAssistant());
+
+    /** @type {import('./assistant_steps_add')} */
+    this._assistantAdd = document.querySelector('assistant-steps-add');
+
+    /** @type {import('./assistant_steps_remove')} */
+    this._assistantRemove = document.querySelector('assistant-steps-remove');
+
+    this._initialized = true;
+  }
+
+  initCallbacks(onAllTranslationsRemoved, onTranslationRemoved) {
+    this._assistantRemove.initCallbacks(onAllTranslationsRemoved, onTranslationRemoved);
+  }
+
+  async openAssistant(moduleType) {
+    await assistantController.initState(moduleType);
+
+    const appContainerWidth = $(window).width() - 10;
+    var wizardWidth = null;
+
+    if (appContainerWidth < 1100) {
+      wizardWidth = appContainerWidth;
+    } else {
+      wizardWidth = 1100;
+    }
+
+    const wizardHeight = $(window).height() * 0.75;
```

Comment:
The ModuleAssistant used to be called "ModuleWizard". I got rid of the term "wizard" some time ago, I would suggest to also replace it here (with assistant).